Client/Nimbus/NimbusManager.swift

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+protocol NimbusManageable { }
+
+extension NimbusManageable {
+    var nimbusManager: NimbusManager {
+        return NimbusManager.shared
+    }
+}
+
+enum NimbusManagerLayers: Equatable {
+    case allLayers
+    case featureFlags
+}
```

Comment:
This might go the way of the dodo based on test results. Ignore for now. Same as the commented out code in the rest of this file.